xpublish_edr/plugin.py

```diff
@@ -80,6 +108,146 @@ def dataset_router(self, deps: Dependencies):
+                    "trs": 'TIMECRS["DateTime",TDATUM["Gregorian Calendar"],CS[TemporalDateTime,1],AXIS["Time (T)",unspecified]]',  # noqa
+                }
+
+            if "Z" in ds_cf:
+                units = ds_cf["Z"].attrs.get("units", "unknown")
+                positive = ds_cf["Z"].attrs.get("positive", "up")
+                elevations = ds_cf["Z"].values
+                min_z = elevations.min()
+                max_z = elevations.max()
+                elevation_values = ",".join([str(e) for e in elevations])
+
+                extents["vertical"] = {
+                    "interval": [
+                        min_z,
+                        max_z,
+                    ],
+                    "values": elevation_values,
+                    "vrs": f"VERTCRS[VERT_CS['unknown'],AXIS['Z',{positive}],UNIT[{units},1]]",  # noqa
+                    "positive": positive,
+                    "units": units,
+                }
+
+            parameters = {
+                k: variable_description(v)
+                for k, v in projected_ds.variables.items()
+                if "axis" not in v.attrs
+            }
+
+            crs_details = [
+                {
+                    "crs": crs.to_string(),
+                    "wkt": crs.to_wkt(),
+                },
+            ]
+
+            # 4326 is always available
+            if crs != DEFAULT_CRS:
+                crs_details.append(
+                    {
+                        "crs": DEFAULT_CRS.to_string(),
+                        "wkt": DEFAULT_CRS.to_wkt(),
+                    },
+                )
+
+            return {
+                "id": id,
+                "title": title,
+                "description": description,
+                "links": [],
+                "extent": extents,
+                "data_queries": {
+                    "position": {
+                        "href": "/edr/position?coords={coords}",
+                        "hreflang": "en",
+                        "rel": "data",
+                        "templated": True,
+                        "variables": {
+                            "title": "Position query",
+                            "description": "Returns position data based on WKT `POINT(lon lat)` or `MULTIPOINT(lon lat, ...)` coordinates",  # noqa
+                            "query_type": "position",
+                            "coords": {
+                                "type": "string",
+                                "description": "WKT `POINT(lon lat)` or `MULTIPOINT(lon lat, ...)` coordinates",  # noqa
+                                "required": True,
+                            },
+                            "output_format": available_output_formats,
+                            "default_output_format": "cf_covjson",
+                            "crs_details": crs_details,
+                        },
+                    },
+                    "area": {
+                        "href": "/edr/area",
```

Comment:
Should this also be:
```suggestion
                        "href": "/edr/area?coords={coords}",
```